guides/source/association_basics.md

```diff
@@ -110,13 +110,37 @@ class CreateBooks < ActiveRecord::Migration[5.0]
 
     create_table :books do |t|
       t.belongs_to :author, index: true
-      t.datetime :published_at
+      t.string :name
       t.timestamps
     end
   end
 end
 ```
 
+As an example, using the rails console, let's first create an author:
+
+```ruby
+ @author = Author.create(name: "David Heinemeier Hansson")
+ => #<Author id: 1, name: "David Heinemeier Hansson", ...>
+```
+
+Now we can associate this author with a book:
+
+```ruby
+@book = Book.create(name: "Remote: Office Not Required", author_id: @author.id)
+ => #<Book id: 1, author_id: 1, name: "Remote: Office Not Required", ...>
+```
+
+Note that we inserted the id of the author David Heinemeier Hansson into the `author_id` field of our book. With this association in place we can view this assocation from the perspective of the book:
```

Comment:
A comma seems more appropriate here after place; as in " With this association in place, we can view this ". No?